ledger/participant-state/kvutils/app/src/main/scala/com/daml/ledger/participant/state/kvutils/app/LedgerFactory.scala

```diff
@@ -93,15 +93,19 @@ trait ConfigProvider[ExtraConfig] {
   def partyConfig(@unused config: Config[ExtraConfig]): PartyConfiguration =
     PartyConfiguration.default
 
-  def initialLedgerConfig(config: Config[ExtraConfig]): InitialLedgerConfiguration =
+  def initialLedgerConfig(config: Config[ExtraConfig]): InitialLedgerConfiguration = {
+    val defaultInitialConfiguration = Configuration.reasonableInitialConfiguration
```

Comment:
Nit: I don't think this binding is worth it; you don't have it for the sandbox.

Reply:
Agree, will remove